examples/Cassie/multibody_sim.cc

```diff
@@ -80,11 +77,20 @@ int do_main(int argc, char* argv[]) {
   if (FLAGS_floating_base) {
     multibody::addFlatTerrain(&plant, &scene_graph, .8, .8);
   }
-  addCassieMultibody(&plant, &scene_graph, FLAGS_floating_base,
-                     "examples/Cassie/urdf/cassie_v2.urdf");
 
+  std::string urdf;
+  if (FLAGS_spring_model) {
+    urdf = "examples/Cassie/urdf/cassie_v2.urdf";
+  } else {
+    urdf = "examples/Cassie/urdf/cassie_fixed_springs.urdf";
+  }
+
+  addCassieMultibody(&plant, &scene_graph, FLAGS_floating_base, urdf,
+      FLAGS_spring_model, true);
```

Comment:
I'm not sure if this is done intentionally. The last should be `FLAGS_spring_model`?

Reply:
Ah never mind, I was misled by the argument name. I thought the it is a flag for the heel leaf springs.

Reply:
Maybe we can change the name to `add_loop_closure` instead of `add_loop_closure_springs`?

Reply:
Done.